docs/program/extend/modular-resources/examples/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Examples of configuring modular resources"
```

Comment:
I think we can potentially just add this to the main page?

Reply:
I do prefer it separate to help with a place for the RPLIDAR & future modular mini-tutorials, but I'm not fixed on it-- @npentrel LMK if you would prefer for it to be moved back

Reply:
I'm not sure what exactly you mean by making sure both tutorials are linked! Do you mean on /program/extend/modular-resources/ or /program/extend/? On both places neither are linked, but I used the canbus picture for the examples page

Reply:
If it helped I could move the examples section & those example cards to the end of the main Modular Resources page. It might get swallowed up in the long page but it might also help it be easier to navigate